core/src/main/java/de/unistuttgart/iaas/amyassist/amy/core/Core.java

```diff
@@ -74,8 +76,13 @@
 	/**
 	 * The method executed by the main method
 	 *
+	 * @param args
+	 *            The arguments for the core.
 	 */
-	void run() {
+	void run(String[] args) {
+		this.cmaHandler = new CommandLineArgumentHandler(args);
+		if (!this.cmaHandler.shouldProgramContinue())
+			return;
```

Comment:
move this in a method called `start`, which calls `run` if the shouldProgramContinue